common-api/src/main/resources/common_api.yaml

```diff
@@ -43,6 +43,11 @@ definitions:
     description: a domain for concepts corresponding to a table in the OMOP schema
     enum: &DOMAIN [OBSERVATION, PROCEDURE, DRUG, CONDITION, MEASUREMENT, DEVICE, DEATH, VISIT]
 
+  SearchType:
```

Comment:
Is this used int he API? If not - I'd suggest just using a normal Java enum instead in ConceptService

Reply:
This is used in the api conceptscontroller too.

Reply:
Sure - but it's not actually used in the Swagger API (e.g. a client would never need to reference this) - it's just an internal implementation detail so I'd  recommend just leaving it in Java.